java/component/keyple-core/src/test/java/org/eclipse/keyple/core/seproxy/plugin/reader/WaitForSeRemovalStateTest.java

```diff
@@ -49,7 +49,7 @@ public void setUp() {
   @Test
   public void waitForRemoval_SINGLESHOT() throws Exception {
     /*
-     * ------------ input polling mode is STOP SE has been removed within timeout
+     * ------------ input polling mode is STOP card has been removed within timeout
      */
     AbstractObservableLocalReader r = AbsSmartInsertionTheadedReaderTest.getMock(READER_NAME);
     WaitForSeRemovalState waitForSeRemovalState = new WaitForSeRemovalState(r);
```

Comment:
oups

Reply:
All names like WaitForSeXyz changed to WaitForCardXyz